src/components/Footer/index.jsx

```diff
@@ -17,19 +17,24 @@ const Footer = ({ classes }) => {
     linkUnderline,
     textWhite,
     divider,
-    callIcon
+    callIcon,
+    footer,
```

Comment:
Not urgent, but perhaps make this something more semantic at some point so that we don't have footer prop in the Footer component (might get a bit confusing when we come back to this)

Reply:
I'll update, thank you :-)